projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -2004,7 +2004,7 @@ public function get_amp_display( $post ) { // phpcs:ignore VariableAnalysis.Code
 	 * @return void
 	 */
 	public function process_request( $post, array $post_data ) {
-		$fb_url = $this->http() . '://www.facebook.com/sharer.php?u=' . rawurlencode( $this->get_share_url( $post->ID ) ) . '&t=' . rawurlencode( $this->get_share_title( $post->ID ) );
+		$fb_url = $this->http() . '://www.facebook.com/sharer/sharer.php?u=' . rawurlencode( $this->get_share_url( $post->ID ) ) . '&t=' . rawurlencode( $this->get_share_title( $post->ID ) );
```

Comment:
Rather than using `$this->http()` to rely on whether or not one is using https, why not just hard-code `https`, since this should be irrelevant to what the WP site is using? I'll note that it's already always `https` in the three other instances of this URL.

```suggestion
		$fb_url = 'https://www.facebook.com/sharer/sharer.php?u=' . rawurlencode( $this->get_share_url( $post->ID ) ) . '&t=' . rawurlencode( $this->get_share_title( $post->ID ) );
```